library/psa_crypto_core.h

```diff
@@ -362,9 +363,10 @@ int psa_key_production_parameters_are_default(
  *       entry point.
  *
  * \param[in]  attributes         The attributes for the key to generate.
- * \param[in]  params             The production parameters from
- *                                psa_generate_key_ext().
- * \param      params_data_length The size of `params->data` in bytes.
+ * \param[in] custom              Custom parameters for the key generation.
+ * \param[in] custom_data         Variable-length data associated with \c custom.
+ * \param custom_data_length
+ *                                Length of `custom_data` in bytes.
```

Comment:
Why not have this on the same line?

Reply:
Likely because the name used to be longer. Anyway, fixed.